srml/council/src/lib.rs

```diff
@@ -14,7 +14,243 @@
+//! less than the threshold.
+//!
+//! #### Council Seats
+//!
+//! - `set_approvals` - Set candidate approvals. Approval slots stay valid as long as candidates in those slots are
+//! registered.
+//! - `proxy_set_approvals` - Set candidate approvals from a proxy. Approval slots stay valid as long as candidates
+//! in those slots are registered.
+//! - `reap_inactive_voter` - Remove a voter. Can be called by anyone who is a voter. At the end of this call,
+//! either the reported or the reportee will get removed.
+//! - `retract_voter` - Remove a voter. All votes are cancelled and the voter deposit is returned.
+//! - `submit_candidacy` - Submit oneself for candidacy. Account must have enough transferrable funds in it to pay
+//! the bond.
+//! - `present_winner` - Claim that the `signed` origin is one of the top candidates.
+//! - `set_desired_seats` - Set the desired council member count.
+//! - `remove_member` - Remove a council member immediately. A tally happens instantly (if not already in a
+//! presentation period) to fill the seat if removal means that the desired members are not met.
+//! - `set_presentation_duration` - Set the presentation duration.
+//! - `set_term_duration` - Set the term duration.
+//!
+//! #### Council Voting
+//!
+//! - `propose` - Propose a council proposal.
+//! - `vote` - Vote on a council proposal by a councillor.
+//! - `veto` - Veto a council proposal by a councillor.
+//! - `set_cooloff_period` - Specify cooling off period.
+//! - `set_voting_period` - Specify voting period.
+//! - `on_finalize` - Signature declaration that runs anything that needs to be done at the end of the block.
+//!
+//! ### Public Functions
+//!
+//! #### Council Motions
+//!
+//! - `is_councillor` - Check if a councillor is a member of the active council.
+//!
+//! #### Council Seats
+//!
+//! - `presentation_active` - Check if we're currently in a presentation period.
+//! - `is_a_candidate` - Check if a specific account id is a registered candidate.
+//! - `next_vote_from` - Determine the block when a vote can occur.
+//! - `next_tally` - The block number when the tally for the next election will occur.
+//!
+//! #### Council Voting
+//!
+//! - `is_vetoed` - Check if a council proposal has been vetoed.
+//! - `will_still_be_councillor_at` - Check each account id of the active council to determine what block number
+//! they will still be active at.
+//! - `is_councillor` - Check if a given account id is a councillor.
+//! - `tally` - The count of the yay and nay votes associated with voting on a council proposal.
+//!
+//!
+//! ### Snippet: Get the next seats tally block number.
+//!
+//!	The next tally of the seats voting will be scheduled based on the condition of needing
+//! more councillors or not. If no councillor is needed, the associated function to get the next tally,
+//! namely `next_tally()` will return `None`. Else, it will return an `Option` containing
```

Comment:
```suggestion
//! namely `next_tally()`, will return `None`.  Else, it will return an `Option` containing
```

Reply:
HTML/MD will render these as single-space. There is not a single modern style guide that recommends two spaces between sentences.

Reply:
> HTML/MD will render these as single-space. There is not a single modern style guide that recommends two spaces between sentences.

I was not aware of the latter, sorry.